src/styles/images/index.md.njk

```diff
@@ -8,53 +8,72 @@ layout: layout-pane.njk
 
-Alternative text, or alt text, is read out by screen readers or displayed if an image does not load or if images have been switched off.
+- a decorative image that does not include important content
+- an icon with a text label, where the alt text would repeat the text label
+- an image used in a link, where the image is not needed to understand the link
 
-All images, except decorative images, must have alt text that:
+To decide if an image needs alt text, see the [Web Accessibility Initiative alt decision tree](https://www.w3.org/WAI/tutorials/images/decision-tree/).
+
+#### Writing good alt text
+
+Good alt text:
 
 - tells people what information the image provides
 - describes the content and function of the image
 - is specific, meaningful and concise
 
 Use normal punctuation, like commas and full stops, so the text is easy to read and understand.
 
-Do not:
+Generally, aim for no more than 2 sentences of alt text.
+
+Do not write alt text that:
 
-- include the name of the photographer or person who created the image
-- start with ‘Image of’, ‘Graphic of’ or ‘Photo of’
-- repeat information from the page
-- include extra information not in the image
+- includes the name of the photographer or person who created the image
+- starts with ‘Image of’, ‘Graphic of’ or ‘Photo of’
+- repeats information from the page
+- includes extra information not in the image
 
 {{ example({group: "styles", item: "images", example: "alt-text", html: true, open: true, size: "l"}) }}
 
-Some images do not need descriptions. Use `alt=""` if the image is:
+### Avoid images that contain text
 
-- decorative and does not include important content
-- an icon that has a text label that the alt text would repeat
-- used in a link and the image is not needed to understand the link
+Text becomes unreadable when the user zooms into images, especially PNGs and JPGs. If the image does contain text, use SVG because it remains readable when the user zooms in. The alt text should be the same as the text displayed in the image.
 
-Use the [Web Accessibility initiative alt decision tree](https://www.w3.org/WAI/tutorials/images/decision-tree/) to work out if the image needs alt text.
+## Aspect ratio
 
-Avoid images that contain text, especially PNGs and JPGs because the text becomes unreadable when the user zooms in. If the image does contain text, use SVG because it remains readable when the user zooms in. The alt text should be the same as the text displayed in the image.
+### Photography
+
+The aspect ratio for photographs should be 3:2.
+
+{{ example({group: "styles", item: "images", example: "default", html: false, open: false, size: "l"}) }}
+
+
+### Illustrations or representative imagery
+
+If your image represents something physical, such as a letter, document or credit card, you should use the aspect ratio of that item.
+
+{{ example({group: "styles", item: "images", example: "representative", html: false, open: false, size: "l"}) }}
```

Comment:
Comment from @36degrees to add alt text to this exampe.

Reply:
@calvin-lau-sig7 Here's a potential alt text draft we can use to get started and iterate upon:
"Illustration of a V11 reminder letter, depicting the light blue letterhead area, a white circle in the top-right corner and a few paragraphs of text"

Reply:
Nice! I've added this to the PR with a minor edit.